.changeset/dull-keys-punch.md

```diff
@@ -0,0 +1,6 @@
+---
+'create-amplify': patch
+'@aws-amplify/backend-data': patch
+---
+
+fix(@aws-amplify/backend-data) - Align authorization modes with client
```

Comment:
who is "client"?

Reply:
Adding clarification which I will post in a moment.